pyxrf/model/load_data_from_db.py

```diff
@@ -1670,6 +1839,26 @@ def write_db_to_hdf_base(fpath, data,
                 raise IOError(f"'write_db_to_hdf_base': File '{fpath}' already exists.")
 
     with h5py.File(fpath, file_open_mode) as f:
+
+        # Create metadata group
+        metadataGrp = f.create_group(f"{interpath}/scan_metadata")
```

Comment:
Is there any particular reason to use the mixed naming conventions, i.e. the camel-case here, and underscored names of variables everywhere else?

Reply:
Which manual? Is there a link?